sound/soc/sof/ipc3-topology.c

```diff
@@ -1587,6 +1587,10 @@ static int sof_ipc3_widget_setup_comp_dai(struct snd_sof_widget *swidget)
 		kfree(config);
 	}
 
+	/* set DAI ops */
+	if (sof_ops(sdev)->set_dai_widget_ops)
+		sof_ops(sdev)->set_dai_widget_ops(sdev, swidget);
```

Comment:
what does 'set the ops' mean?

usually we have set/get, but there's no get?

did you mean 'assign_dai_widget_ops'? or 'select_dai_widget_ops'?